src/extension.ts

```diff
@@ -2,6 +2,8 @@
 // All rights reserved.
 
 import * as vscode from 'vscode';
+import { homedir } from 'os';
+import { join } from 'path';
```

Comment:
Is it better to import just 'join' or rather the entire lib?

Reply:
If you only need the function and nothing else, like here, then this is better, yes. You often also want type declarations provided by a module for type checking - but here this is better.